src/Microsoft.Extensions.ServiceDiscovery.Abstractions/ServiceEndpoint.cs

```diff
@@ -11,35 +11,35 @@ namespace Microsoft.Extensions.ServiceDiscovery;
 /// <summary>
 /// Represents an endpoint for a service.
 /// </summary>
-[DebuggerDisplay("{GetEndPointString(),nq}")]
-public abstract class ServiceEndPoint
+[DebuggerDisplay("{GetEndpointString(),nq}")]
+public abstract class ServiceEndpoint
 {
     /// <summary>
     /// Gets the endpoint.
     /// </summary>
-    public abstract EndPoint EndPoint { get; }
+    public abstract EndPoint Endpoint { get; }
 
     /// <summary>
     /// Gets the collection of endpoint features.
     /// </summary>
     public abstract IFeatureCollection Features { get; }
 
     /// <summary>
-    /// Creates a new <see cref="ServiceEndPoint"/>.
+    /// Creates a new <see cref="ServiceEndpoint"/>.
     /// </summary>
-    /// <param name="endPoint">The endpoint being represented.</param>
+    /// <param name="endpoint">The endpoint being represented.</param>
     /// <param name="features">Features of the endpoint.</param>
-    /// <returns>A newly initialized <see cref="ServiceEndPoint"/>.</returns>
-    public static ServiceEndPoint Create(EndPoint endPoint, IFeatureCollection? features = null) => new ServiceEndPointImpl(endPoint, features);
+    /// <returns>A newly initialized <see cref="ServiceEndpoint"/>.</returns>
+    public static ServiceEndpoint Create(EndPoint endpoint, IFeatureCollection? features = null) => new ServiceEndpointImpl(endpoint, features);
 
     /// <summary>
-    /// Gets a string representation of the <see cref="EndPoint"/>.
+    /// Gets a string representation of the <see cref="Endpoint"/>.
     /// </summary>
-    /// <returns>A string representation of the <see cref="EndPoint"/>.</returns>
-    public virtual string GetEndPointString() => EndPoint switch
+    /// <returns>A string representation of the <see cref="Endpoint"/>.</returns>
+    public virtual string GetEndpointString() => Endpoint switch
```

Comment:
EndPoint?

Reply:
The idea in API review was that names referencing a ServiceEndpoint uses the new capitalization scheme, but names referring to a System.Net.EndPoint still do it the old way.

We noticed this in the `public abstract EndPoint EndPoint { get; }` property, but not for the `Create(EndPoint endpoint` parameter which should probably be `endPoint` even though that wasn't what we explicitly approved in API review.

I think `Endpoint` is fine here since it's referring to `ServiceEndpoint`, but now I'm wondering why we approved this method to begin with. We decided to get rid of `ServiceEndPointQuery.OriginalString` because we decided `ToString()` was sufficient and we could always add the `OriginalString` parameter later. I feel the same logic applies here.

Can we just delete this? We can send out an email and update the API approval if everyone is on board.

Reply:
Yes, we can delete this and use ToString()